packages/desktop-client/src/components/Settings.js

```diff
@@ -428,9 +428,28 @@ function FileSettings({
         </View>
       </View>
 
-      <View style={{ marginTop: 30, alignItems: 'flex-start' }}>
+      <View style={{ marginTop: 30, maxWidth: 600, alignItems: 'flex-start' }}>
+        <Title name="Import" />
+        <Text style={{ lineHeight: '1.4em' }}>
+          Importing creates a new budget file and must be done after you are
+          signed out from the server. Once signed out, there is an option to
+          import a file.
+        </Text>
+        <Button style={{ marginTop: 10 }} onClick={() => signOut()}>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
        <Button style={{ marginTop: 10 }} onClick={signOut}>
```